README.md

```diff
@@ -1,31 +1,249 @@
 # Limestone API
 
-An api to access trusted token pricing data secured on Arweave and protected by provider's collateral.
+Limestone API is a Javascript library for fetching trusted token pricing data from [Limestone data ecosystem](https://github.com/limestone-finance/limestone/blob/master/README.md).
 
-## Usage
+It is a Javascript wrapper for [Limestone HTTP Api](docs/HTTP_API.md).
 
+## ✅ Why Limestone API
+### ✓ Secure
+Limestone pricing data is secured on Arweave and protected by provider's collateral.  
+[Learn more](https://github.com/limestone-finance/limestone/blob/master/README.md)
+
+### ✓ Easy to use
+You don't need any API keys. Just install npm package and add a single line of code.  
+[Quick start](docs/QUICK_START.md)
+
+### ✓ Free
+Limestone API is absolutely free.
+
+### ✓ TypeScript Support
+Limestone API is fully written in Typescript and then compiled to JavaScript.  
+[Source code](https://github.com/limestone-finance/limestone-api)
+
+## 📦 Installation
+
+### Using npm
+```bash
+npm install limestone-api
+```
+
+### Using yarn
+```bash
+yarn add limestone-api
 ```
-const limestone = require('@limestone/api');
 
-const price = limestone.getPrice("AR");
+## 🤖 Usage
+
+### Importing
+
+```js
+// Using Node.js `require()`
+const limestone = require('limestone-api');
+
+// Using ES6 imports
+import limestone from 'limestone-api';
 
-console.log(price.value);
 ```
 
-## Data format
+### Get latest price for a single currency
+```js
+const price = await limestone.getPrice("AR");
+
+console.log(price.value); // latest price value for AR token (in USD)
+console.log(price.timestamp); // timestamp for the price
+```
+💡 Note: All price values are in USD. You can fetch price data for BTC, ETH, AR, EUR and any other of [ 158 supported currencies.](docs/ALL_SUPPORTED_CURRENCIES.md)
+
+<br/>
+
+<details>
+  <summary>Price data format</summary>
 
-- TODO: udpate data format
+  ```js
+  {
+    value: 123.23, // Number: Price value in USD
+    timestamp: 1617146511173, // Number: Timestamp (ms) for price
+    provider: "I-5rWUehEv-MjdK9gFw09RxfSLQX9DIHxG614Wf8qo0", // String: Provider arweave address
+    permawebTx: "V8FUU0BG4kVOJwKWHzgkn1aEFm-eanhqqEXfPFY7pmI", // String: Arweave transaction id
+    version: "0.2", // String: Limestone version
+    source: {"coingecko": 123,"sushiswap": 123.23,"uniswap": 123.35}, // Object: Price from different sources
+    signature: "KWHzgkn1a...", // String: price signature
+    providerPublicKey: "fPFY7pm..." // String: Provider public key
```

Comment:
Hmm, do we need to return it to client - it's very technical and could be misleading